VaticanBAV/et/BAVet116.xml

```diff
@@ -67,132 +71,150 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
-                          </msItem>
-                          <msItem xml:id="ms_i1.16">
-                            <locus from="153r" to="160v"/>
-                            <title type="complete" ref="LIT1960Mashaf#AnaphoraJohnChrysostom"/>
-                          </msItem>
-                          <msItem xml:id="ms_i1.17">
-                            <locus from="160v" to="164r"/>
-                            <title type="complete" ref="LIT1960Mashaf#AnaphoraDioscorus"/>
-                          </msItem>
-                          <msItem xml:id="ms_i1.18">
-                            <locus from="164r" to="171v"/>
-                            <title type="complete" ref="LIT1960Mashaf#AnaphoraJacobSerug"/>
-                          </msItem>
-                          <msItem xml:id="ms_i1.19">
-                            <locus from="171v" to="181r"/>
-                            <title type="complete" ref="LIT1098Anapho"/>
-                          </msItem>
-                          <msItem xml:id="ms_i1.20">
-                            <locus from="182v" to="186v"/>
-                            <title type="complete" ref="LIT2038Naakkw"/>
-                          </msItem>
-                          <msItem xml:id="ms_i1.21">
-                            <locus from="186v" to="194r"/>
-                            <title type="complete" ref="LIT1569habeka"/>
+
+                        </msItem>
+
+                        <msItem xml:id="ms_i1.2">
+                          <locus from="20v" to="24r"/>
+
+                          <msItem xml:id="ms_i1.2.1">
+                            <locus from="20v" to="22v"/>
+                            <title>Prayers of supplication for the Deceased</title>
                           </msItem>
-                          <msItem xml:id="ms_i1.22">
-                            <locus from="194r" to="197v"/>
-                            <title type="complete" ref="LIT2038Naakkw"/>
+
+                          <msItem xml:id="ms_i1.2.2">
+                            <locus from="22v" to="24r"/>
+                            <title>መስተብቍዕ፡</title>
                             <note>
-                              Repeated; added at the end is the prayer of the laying on of hands.
+                              Prayers before the reading of the Miracles of Our Lady Mary
                             </note>
                           </msItem>
+
+                        </msItem>
+
+                        <msItem xml:id="ms_i1.3">
+                          <locus from="24r" to="29r"/>
+                          <title type="complete" ref="LIT2605Zayena"/>
+                          <note>With the litany written underneath</note>
                         </msItem>
+
+                        <msItem xml:id="ms_i1.4">
+                          <locus from="29v" to="69v"/>
+                          <title type="complete" ref="LIT1960Mashaf#SerataQeddase"/>
```

Comment:
the complete Serata Qeddase? and only one of the prayers highlighted?

Reply:
That's, unfortunately, basically all that's given in the catalogue.

Reply:
OK